state-sync/diem-data-client/src/diemnet/metrics.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use diem_crypto::_once_cell::sync::Lazy;
+use diem_metrics::{
+    register_histogram_vec, register_int_counter_vec, HistogramTimer, HistogramVec, IntCounterVec,
+};
+
+pub const TOTAL: &str = "TOTAL";
```

Comment:
nit: maybe name this `TOTAL_COUNT_LABEL` and add a comment above it? e.g., the comment you added to `increment_counter` could be moved here

Reply:
Updated name: `pub const TOTAL_COUNT_LABEL: &str = "TOTAL_COUNT";`